data/nlu/nlu.md

```diff
@@ -4506,6 +4550,10 @@
 - i dont like bots
 - this is a really frustrating experience
 - you're rather dull
+- Thats so rude
+- sadly
+- That tool here isnt good
+- your crazy
```

Comment:
last one probably an insult

Reply:
yes i don't think i added this one 🤔

Reply:
oh i know, i'm also just re-reviewing all the changes in general, i do it when i push my own changes 😄